Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/PaloAltoNetworksIoT3rdParty.py

```diff
@@ -5,11 +5,11 @@
 urllib3.disable_warnings()
 
 DEFAULT_HEADERS = {
-    'X-Access-Key': demisto.params().get("Access Key"),
-    'X-Key-Id': demisto.params().get("Key ID"),
+    'X-Access-Key': demisto.params().get('customer_id_creds', {}).get('password') or demisto.params().get("Access Key"),
```

Comment:
Access Key and Key ID seem like should be grouped under 1 creds object. lets have Customer ID as the seperate one (double check this)

Reply:
youre right, ill change